docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
```

Comment:
The reason for the error is the plugindefinition states what is valid, and there is a mismatch.